lib/builder/webpack/base.js

```diff
@@ -49,7 +49,7 @@ export default class WebpackBaseConfig {
     // Don't use hashes when watching
     // https://github.com/webpack/webpack/issues/1914#issuecomment-174171709
     if (this.options.dev) {
-      fileName = fileName.replace(/\[(chunkhash|contenthash|hash)\]\./g, '')
+      fileName = fileName.replace(/\[(chunkhash|contenthash|hash)(?::(\d+))?\]\./g, '')
```

Comment:
Thank you for the pr, good idea.
I think (:\d+)? should be fine here, what do you think?

Reply:
This is copied from the official webpack implementation.

Reply:
Could pelase show me the link?

Reply:
https://github.com/webpack/webpack/blob/8e6a012dbbb1526db1da753b61c43e8c61b3379f/lib/TemplatedPathPlugin.js#L7